src/commands/switch.ts

```diff
@@ -6,7 +6,7 @@ import authSwitch from '../modules/auth/switch'
 export default class Switch extends CustomCommand {
   static description = 'Switch to another VTEX account'
 
-  static examples = ['vtex switch storecomponents', 'vtex switch storecomponents myworkspace']
+  static examples = ['vtex switch storecomponents', 'vtex switch storecomponents/myworkspace']
```

Comment:
I think this syntax of using `account/workspace` is kinda shadowy - an argument is becoming two 
I think we should document on examples the way using flags: `vtex switch storecomponents -w myWorkspace`